cli/root.go

```diff
@@ -213,12 +213,22 @@ func versionCmd() *cobra.Command {
 		Short: "Show coder version",
 		RunE: func(cmd *cobra.Command, args []string) error {
 			var str strings.Builder
-			_, _ = str.WriteString(fmt.Sprintf("Coder %s", buildinfo.Version()))
+			_, _ = str.WriteString("Coder ")
+			if buildinfo.IsAGPL() {
+				_, _ = str.WriteString("(AGPL) ")
```

Comment:
Adding another column breaks version parsing from the binary. Not sure if it'll be a real problem for anyone though.

Reply:
It'd only affect users using the AGPL binary, which I believe won't be a significant portion of users.